rero_ils/dojson/utils.py

```diff
@@ -1255,6 +1267,9 @@ def init_country(self):
                 self.country = self.field_008_data[15:18].rstrip()
             except Exception as err:
                 pass
+        # Use equivalent if country code is obsolete
+        if self.country in _OBSOLETE_COUNTRIES_MAPPING:
```

Comment:
Could also be written like this:
```python
if obsolete_country := _OBSOLETE_COUNTRIES_MAPPING.get(self.country):
    self.country = obsolete_country
```

Reply:
You are shorter version : `self.country = _OBSOLETE_COUNTRIES_MAPPING.get(self.country, self.country)` ;-)